strapi/strapi
# Add a `strapi migrate` command, optionally skip migration on startup

### What does it do?

- Adds a `strapi migrate` command, which runs schema sync and any user migrations
- Adds `--no-migrate` option to the `start` command: `strapi start --no-migrate`

### Why is it needed?

Having a standalone migration command is useful in some environments where the application DB user does not have elevated permissions to add/drop tables, modify columns, etc.

It also allows DB changes to be made without starting a web server.

Addresses a `TODO` comment (removed in PR diff).

### How to test it?

In the `examples/gettingstarted`, run the following commands:

```
npm start // should run migrations
npm start -- --no-migration // should not run migrations
```

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
